crates/aptos-compression/src/lib.rs

```diff
@@ -39,6 +40,7 @@ pub struct CompressionError(String);
 pub fn compress(
     raw_data: Vec<u8>,
     client: CompressionClient,
+    max_bytes: usize,
```

Comment:
Given that we're only using compression for network messages, I'd just have this hard coded to `MAX_APPLICATION_MESSAGE_SIZE` and not dirty the call sites.